theano/tests/breakpoint.py

```diff
@@ -12,8 +12,8 @@ class PdbBreakpoint(Op):
     conditional breakpoint, inside a theano function, based on a symbolic
     scalar condition.
 
-    @type name: String
-    @param name: name of the conditional breakpoint. To be printed when the
+    :type name: String
+    :param name: name of the conditional breakpoint. To be printed when the
```

Comment:
From memory, the first just didn't render and this one does. This PR make Theano better then the current version and do not bring long term problem. So I won't get bugged by this. We didn't call for a rewrite of all the docstring to a specific format.